CHANGELOG.md

```diff
@@ -3,6 +3,85 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.22.4
+
+_Nov 28, 2024_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 📚 Documentation improvements
+- 🌍 Improve Dutch, Spanish, Swedish, Portuguese, Chinese locales on the Data Grid and Date Pickers.
+- 🐞 Bugfixes
+
+Special thanks go out to the community contributors who have helped make this release possible:
+@dloeda, @headironc, @mathzdev, @nphmuller.
+Following are all team members who have contributed to this release:
+@oliviertassinari, @arminmeh, @KenanYusuf.
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@v7.22.4`
+
+- [docs][data grid] Add documentation for the list view feature (#15344) (#15635) @KenanYusuf
```

Comment:
Should move to the docs and remove `[data grid]` (we might want to discuss how to correctly prefix the `[docs]` PRs btw, the more product we have, the weirder it is to group the docs PR together instead of moving them to their respective packages IMHO